app/home/-components/support-section/template.hbs

```diff
@@ -7,6 +7,19 @@
             {{t 'new-home.support-section.header'}}
         </h1>
     </header>
+    <div local-class='flex-container'>
```

Comment:
And would recommend changing the local-class here as well, per the flex-row comment. This echoes of Bootstrap-style nomenclature, which we are trending away from.